ember-cli-build.js

```diff
@@ -45,8 +46,7 @@ module.exports = function(defaults) {
     app.import(path.join(app.bowerDirectory, 'd3/d3.js'));
     app.import(path.join(app.bowerDirectory, 'osf-style/css/base.css'));
     app.import('vendor/assets/ember-osf.css');
-    app.import({
-        test: path.join(app.bowerDirectory, 'ember/ember-template-compiler.js')
-    });
+    app.import(path.join(app.bowerDirectory, 'ember/ember-template-compiler.js'));
+    app.import(path.join(app.bowerDirectory, 'jquery-mockjax/dist/jquery.mockjax.js'));
```

Comment:
Does this add mockjax to our production builds? I think we want it only in development.